### PR TITLE
python: Add SSL to example

### PR DESCRIPTION
**Pull request by hfiguiere**

### CODE REVIEW

File: webapp/python/README.md [hunk 1]
```diff
@@ -15,29 +20,54 @@
...
+
+Then set the environment to indicate where to load the certificate from.
+
+- `SSL_KEY_FILE` contains the path to the private key. If you used
+  the `minica` tool mentionned above, it's the path to the
+  `minica-key.pem` file.
+- `SSL_CRT_FILE` contains the path to the public certificate. If you used
+  the `minica` tool mentionned above, it's the path to the
+  `minica.pem` file.
+- `DISABLE_TLS_CERT_VALIDATION` set to `1` if you use certificates that do
```
Comment (meven) [inline webapp/python/README.md:69]: Could be in `webapp/python/run_https.sh` or derived from `settings.DEBUG`?

File: webapp/python/README.md [hunk 2]
```diff
@@ -15,29 +20,54 @@
...
 2. Start your browser and make it point to the url `http://<host>:8000`
-   
-   To make it reachable by the Collabora Online server use as `<host>` the IP address of the machine where the Django 
-   server is running. In case the Django server can't be reached you could also need to open the port 8000 on the firewall. 
-   
+
+Or with HTTPS after configuring the certificates (see below)
+
+1. Start the Django development server: `run_https.sh`
+2. Start your browser and make it point to the url `https://<host>:8000`
```
Comment (meven) [inline webapp/python/README.md:32]: How will COOL allow talking with this server with a self-signed certificate.
I don't know myself.
We should probably mention something about it.

Comment (hfiguiere) [reply webapp/python/README.md:32]: I stashed my patche and did `make` && `make run` and it works. I think in debug it doesn't verify. That's the baseline. Also it's all `localhost`.

File: webapp/python/README.md (hunk 1 above)
Comment (hfiguiere) [reply webapp/python/README.md:69]: actually I don't want to code this anywhere. It has to be done by the user deliberately.

I use `direnv` for that purpose. It's the same with the Node examples.

**Review by meven — APPROVED** [review]

**Pull request merged**